salt/_states/metalk8s_volumes.py

```diff
@@ -0,0 +1,125 @@
+# coding: utf-8
+'''State module to deal with MetalK8s Volume.'''
+
+import logging
+
+from salt.exceptions import CommandExecutionError
+
+log = logging.getLogger(__name__)
+
+
+__virtualname__ = 'metalk8s_volumes'
+
+
+def __virtual__():
+    return __virtualname__
+
+
+def present(name, volume):
+    """Ensure that the backing storage exists for the specified volume.
+
+    Args:
+        name   (str): SLS caller name
+        volume (str): Volume name
```

Comment:
It's not common salt practice in salt normaly name is something usefull and mandatory for state, it's not only a "name"

For me you should remove the volume argument and use name as volume name

Reply:
(And it's the same for other function of this file)

Reply:
Ho thanks for the explanation, yeah I'll get rid of the `volume` parameter and use name instead!

Reply:
Done